src/App/App.jsx

```diff
@@ -50,11 +50,6 @@ function App() {
   //const [token, setToken] = ...
 
   const handleAddToCart = (item) => {
-    // if (!cart) {
-    //   createCart(cart).then((res) => {
-    //     console.log("cart created");
-    //   });
-    // }
     addToCart(item)
       .then((res) => {
         console.log("item added", res.cart);
```

Comment:
Please, add `closeActiveModal();` in line 55 to close the modal in `.then` rather than in line 63